app/models/collection.rb

```diff
@@ -3,7 +3,15 @@
 # Web archive collection
 class Collection < ApplicationRecord
   self.primary_key = 'druid'
-  validates_presence_of :title, :druid, :active, :embargo_months
+  validates_presence_of :title, :druid, :active, :embargo_months, :wasapi_provider_account
   validates :druid, format: { with: /druid:.+/,
                               message: 'must begin with druid:' }
+
+  def wasapi_provider_account
+    wasapi_provider && wasapi_account ? "#{wasapi_provider}:#{wasapi_account}" : nil
+  end
+
+  def wasapi_provider_account=(wasapi_provider_account)
```

Comment:
can you use a different variable name that isn't the same as a method?

Reply:
Done.